### ISSUE TITLE
Draggable rows also does a drag to copy

### ISSUE THREAD

**Issue by michaeljacobdavis** [adazzle/react-data-grid#463]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/adazzle/react-data-grid/blob/master/CONTRIBUTING.md
```

**Current behavior**
The selected cells value is duplicated across all rows that are dragged over.
![](http://g.recordit.co/Rmr9mqoDZk.gif)

**Expected/desired behavior**
The row is dragged and the cell is deselected.

**Reproduction of the problem**
If the current behavior is a bug or you can illustrate your feature request better with an example, please provide the steps to reproduce and if possible a minimal demo of the problem.
This [Gist](https://gist.github.com/michaeljacobdavis/bac988bee0603db970f0cbb2cf961d15) recreates the problem when pasted into the examples editor.

Basically it's when you have editable/draggable cells and draggable rows enabled.

**Solution**
My first thought is to add a check in [`onDragStart`](https://github.com/adazzle/react-data-grid/blob/master/src/ReactDataGrid.js#L312) that checks if *what* is being dragged, but I'm not super familiar with the internals of react-data-grid, so I'd love some guidance on fixing this. I have no problem doing a PR for it.

**Comment by diogofcunha** [comment]
Hi @michaeljacobdavis, thanks for the feedback.

That bug was fixed in 1.0.43.

**Comment by michaeljacobdavis** [comment]
@diogofcunha I can reproduce this in master currently. Please checkout this [Gist](https://gist.github.com/michaeljacobdavis/bac988bee0603db970f0cbb2cf961d15) and paste it into one of the live examples http://adazzle.github.io/react-data-grid/examples.html#/row-reordering.

**Comment by michaeljacobdavis** [comment]
@diogofcunha This is still an issue. Can you reopen?
